.github/workflows/helm-multi-release.yaml

```diff
@@ -0,0 +1,200 @@
+        type: string
+      project-root-dir:
+        description: "Github  dir where the charts dir is located"
+        required: false
+        default: "."
+        type: string
+      helm-version:
+        description: "The Helm version."
+        required: false
+        default: "v3.10.1"
+        type: string
+      lint-config-path:
+        description: "The path to the lint configuration file (See https://github.com/helm/chart-testing/blob/main/pkg/config/test_config.yaml)."
+        required: false
+        default: ".github/lint-config.yaml"
+        type: string
+
+    secrets:
+      github-username:
+        description: "The GitHub username for committing the changes."
+        required: true
+      github-email:
+        description: "The GitHub email for committing the changes."
+        required: true
+      github-token:
+        description: "The GitHub token for committing the changes."
+        required: true
+
+jobs:
+  helm-lint:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Lint Helm chart
+        uses: bakdata/ci-templates/actions/helm-lint@v1.0.0
+        with:
+          lint-config-path: "${{ inputs.lint-config-path }}"
+
+  update-version:
+    runs-on: ubuntu-22.04
+    needs: [helm-lint]
+    env:
+      root-dir: ${{ inputs.charts-dir }}
+      ref: ${{ inputs.root-branch }}
+      release-tag: ${{github.ref_name}}
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.ref }}
+
+      - name: Use the tag to update chart.yaml files
+        run: |
+          sudo add-apt-repository ppa:rmescandon/yq
+          sudo apt update
+          sudo apt install yq -y
+          for subdir in */; do
+            cat "$subdir/Chart.yaml"
+            yq e -i '.version = "${{  env.release-tag }}"' "$subdir/Chart.yaml"
+            cat "$subdir/Chart.yaml"
+          done
+        shell: bash
+        working-directory: ${{ env.root-dir }}
+
+      - name: Commit and push updated charts
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.6.0
+        with:
+          add-untracked: true
+          ref: ${{ env.ref }}
+          commit-message: "Update Chart versions"
+          github-username: ${{ secrets.github-username }}
+          github-email: ${{ secrets.github-email }}
+          github-token: ${{ secrets.github-token }}
+
+  package-new-artifacts:
+    strategy:
+      matrix:
+        subdir: ${{ fromJson(inputs.subdirs) }}
+    runs-on: ubuntu-22.04
+    needs: [update-version]
+    env:
+      charts-subdir: "${{ matrix.subdir }}"
+      root-dir: ${{ inputs.charts-dir }}
+      artifact-dir: "${{ inputs.artifact-dir }}"
+      ref: ${{ inputs.root-branch }}
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.ref }}
+
+      - name: Prepare artifacts
+        run: |
+          cd ${{ env.charts-subdir }}
+          helm package .
+          cd ../../
```

Comment:
You can just use the `-d` flag after you created the folder to provide a destination for the tgz file without changing folders